Detectors/gconfig/g4Config.C

```diff
@@ -39,7 +39,7 @@ void Config()
 /// character: eg. stepLimit+specialCuts.
 
    //Geant4 VMC 3.x
-   Bool_t mtMode = false;
+   Bool_t mtMode = FairRunSim::Instance()->IsMT();
```

Comment:
why not just accessing `confref` here?

Reply:
As fairRunSim class holds alredy the selection of MC, so it seemed to me the best plece where to put this one